File: data/code/lib/clean_data.R
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

### DEFINE
def main():
    df = pd.read_csv('output/data_merged.csv')
    plot_data(df)
    df = clean_data(df)
    df.to_csv('output/data_cleaned.csv')

def plot_data(df):
    plt.hist(df['chips_sold'])
    plt.savefig('output/chips_sold.pdf')

def clean_data(df):
    df[df['chips_sold'] == -999999] = np.NaN
    return(df)
    
### EXECUTE
main()

File: 1_data/source/clean_data.py
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

# Paths
output='../output'

df = pd.read_csv(f'{output}/data_merged.csv')
df.loc[df['chips_sold'] == -999999, 'chips_sold'] = np.nan
df.to_csv(f'{output}/data_cleaned.csv', index = False)
